src/web/app/pages-student/student-profile-page/upload-edit-profile-picture-modal/upload-edit-profile-picture-modal.component.ts

```diff
@@ -65,8 +73,12 @@ export class UploadEditProfilePictureModalComponent implements OnInit {
     this.imageChangedEvent = event;
 
     const file: File = event.target.files[0];
-    if (file) {
+    if (this.validProfileFileTypes.split(', ').includes(file.type)) {
```

Comment:
Rather than splitting here, why not define the valid types as array, then do the `.join(', ')` in the front-end?

Reply:
Makes sense, as HTML can also accept the array string as an argument. I've made the relevant changes, please take a look. Thanks!